src/main/scala/dpla/ingestion3/mappers/providers/CdlExtractor.scala

```diff
@@ -42,12 +43,13 @@ class CdlExtractor extends Extractor with JsonExtractionUtils {
         uri = providerUri(json)
       ),
       OreAggregation(
-        uri = new URI("http://example.com"), //TODO: our url
+        uri = new URI(List(DplaMapData.baseItemUri, mintItemId(rawData)).mkString("")),
```

Comment:
I think this deserves it's own function, or the mkString call should be moved into mintItemId.